provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -1039,6 +1078,12 @@ func validateNoImmutableAttChange(state, plan *ClusterRosaClassicState) diag.Dia
 	common.ValidateStateAndPlanEquals(state.MultiAZ, plan.MultiAZ, "multi_az", &diags)
 	common.ValidateStateAndPlanEquals(state.WorkerDiskSize, plan.WorkerDiskSize, "worker_disk_size", &diags)
 
+	// cluster admin attributes
+	common.ValidateStateAndPlanEquals(state.CreateAdminUser, plan.CreateAdminUser, "create_admin_user", &diags)
+	if !adminCredentialsEqual(state.AdminCredentials, plan.AdminCredentials) {
```

Comment:
I think you can use the ValidateStateAndPlanEquals for that since it verified `!stateAttr.Equal(planAttr) ` please check it and remove it if it already covered....